src/protocols/aeternity/views/AccountDetailsTransactions.vue

```diff
@@ -118,35 +121,54 @@ export default defineComponent({
       }, 200);
     }
 
+    function onViewDidLeaveHandler() {
+      if (polling) {
+        clearInterval(polling);
+      }
+      setScrollConf(false);
+      isDestroyed.value = true;
+    }
+
     watch(
       appInnerScrollTop,
       (value) => {
         setScrollConf(value >= FIXED_TABS_SCROLL_HEIGHT);
       },
     );
 
-    onMounted(() => {
-      initViewport(appInnerElem.value!);
-      if (innerScrollElem.value && appInnerElem.value) {
-        appInnerElem.value.addEventListener('scroll', throttledScroll());
-      }
-      fetchTransactionList();
-      polling = setInterval(() => {
-        if (isAppActive.value) {
-          fetchTransactionList(true);
+    watch(
+      () => props.ionicLifecycleStatus,
+      () => {
+        if (props.ionicLifecycleStatus === 'willEnter') {
+          // reset state since component might not have been unmounted
+          loading.value = false;
+          isDestroyed.value = false;
+          setScrollConf(false);
+          return;
         }
-      }, 10000);
-    });
 
-    onUnmounted(() => {
-      if (polling) {
-        clearInterval(polling);
-      }
-      isDestroyed.value = true;
-    });
+        if (props.ionicLifecycleStatus === 'didEnter') {
+          initViewport(appInnerElem.value!);
+          if (innerScrollElem.value && appInnerElem.value) {
+            appInnerElem.value.addEventListener('scroll', throttledScroll());
+          }
+          fetchTransactionList();
+          polling = setInterval(() => {
+            if (isAppActive.value) {
+              fetchTransactionList(true);
+            }
+          }, 10000);
+          return;
+        }
 
-    onIonViewWillEnter(() => {
-      setScrollConf(false);
+        if (props.ionicLifecycleStatus === 'didLeave') {
+          onViewDidLeaveHandler();
+        }
+      },
+    );
+
+    onUnmounted(() => {
```

Comment:
`onUnmounted` is required because if you change tabs `onIonViewDidLeave` will be triggered but not  `onUnmounted`, but if you close the account-details with the x button only `onUnmounted` will be called and not  `onIonViewDidLeave`